h2/src/main/org/h2/jdbc/JdbcConnection.java

```diff
@@ -1130,79 +1134,102 @@ public PreparedStatement prepareStatement(String sql, int resultSetType,
             checkClosed();
             sql = translateSQL(sql);
             return new JdbcPreparedStatement(this, sql, id, resultSetType,
-                    resultSetConcurrency, false);
+                    resultSetConcurrency, false, false);
         } catch (Exception e) {
             throw logAndConvert(e);
         }
     }
 
     /**
-     * Creates a new prepared statement. This method just calls
-     * prepareStatement(String sql) internally. The method getGeneratedKeys only
-     * supports one column.
+     * Creates a new prepared statement.
      *
      * @param sql the SQL statement
-     * @param autoGeneratedKeys ignored
+     * @param autoGeneratedKeys
+     *            {@link Statement.RETURN_GENERATED_KEYS} if generated keys should
```

Comment:
Yes, they should. It's fixed too.

I can't remove documentation because full build of H2 database for release will check that javadoc comments and tags are in place.